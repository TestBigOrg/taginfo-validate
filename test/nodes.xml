<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="CGImap 0.0.2">
    <bounds minlat="54.0889580" minlon="12.2487570" maxlat="54.0913900" maxlon="12.2524800"/>
    <node id="1" lat="54.0901746" lon="12.2482632" user="pikachu" uid="46882" visible="true" version="1" changeset="676636" timestamp="2008-09-21T21:37:45Z">
    <tag k="cat" v="traffic_signals"/>
    </node>
    <node id="5" lat="54.0902846" lon="12.2482731" user="pikachu" uid="46882" visible="true" version="1" changeset="676636" timestamp="2008-09-21T21:37:45Z">
    <tag k="highway" v="traffic_signals"/>
    </node>

    <node id="2" lat="54.0906309" lon="12.2441924" user="mj" uid="36744" visible="true" version="1" changeset="323878" timestamp="2008-05-03T13:39:23Z">
    <tag k="barrier" v="city_limit"/>
    </node>

    <node id="3" version="1" changeset="12370172" lat="54.0900666" lon="12.2539381" user="hagel" uid="75625" visible="true" timestamp="2012-07-20T09:43:19Z">
    <tag k="width" v="10"/>
    </node>
</osm>

