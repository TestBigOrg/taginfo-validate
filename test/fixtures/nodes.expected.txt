# valid found tags, expected on any object type:
node width=10
# Way tags missing in provided pbf
# Node tags missing in provided pbf
node highway=spacejam
node spacejam=""
# Relation tags missing in provided pbf
# Area tags missing in provided pbf
# Tags with no object type specified missing in provided pbf
